packages/@ourworldindata/utils/src/Util.ts

```diff
@@ -1286,25 +1291,6 @@ export const getArticleFromJSON = (
     }
 }
 
-export function recursivelyMapArticleBlock(
```

Comment:
I had written two functions that did the same thing, so I kept the one that also traversed spans.